content/docs/studio/user-guide/projects-and-experiments/live-metrics.md

```diff
@@ -0,0 +1,102 @@
+# Generate live (real-time) metrics for experiments
+
+In your model training CI action, you can use [DVCLive] to send live updates to
+metrics and plots back to Iterative Studio, without writing them to your Git
+repository. This requires a 3-step process:
+
+1. [Set up an access token](#set-up-an-access-token)
+2. [Configure your model training CI job](#configure-your-model-training-ci-job)
+3. [Send and view the updates](#send-the-updates)
```

Comment:
**View live metrics** is a separate H2 now.

Reply:
Yes. But the 3 steps are about **sending** live updates. So I didn't include **View live metrics** as a separate step here. This list is not the table of contents in this page. But let me know if you think this must be changed.